front/src/styles/scss/applications/rollingStockEditor/_rollingStockForm.scss

```
@@ -295,4 +295,13 @@
       }
     }
   }
+
+  .warning-message {
+    border: 2px solid var(--yellow);
+    border-radius: 0.3rem;
+  }
+
+  .warning-icon {
+    color: var(--yellow);
```

You can move this class in the warning-message class (I think)
```suggestion
    .warning-icon {
      color: var(--yellow);
    }
```

---

fixed